addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -220,6 +255,76 @@ function ComposerFactory({ Entity }) {
             return [...new Set(allPartnerIds)];
         }
 
+        _getFirstMentionSuggestion() {
+            this.active_id = this.mentionSuggestions[0].id;
+        }
+
+        _getLastMentionSuggestion() {
+            this.active_id = this.mentionSuggestions[this.mentionSuggestions.length - 1].id;
+        }
+
+        async _getMentionSuggestions(mentionWord) {
+            this.active_id = 0;
```

Comment:
This probably needs to be updated after the result of the RPC to avoid interface becoming unsync/flicker while the RPC is pending.

Reply:
Done